.npmrc

```diff
@@ -1,2 +1 @@
-engine-strict=true
-ignore_scripts=true
\ No newline at end of file
+engine-strict=true
```

Comment:
@GCHQ-Developer-530 could you open an issue explaining what needs to be run postinstall and I'll create a self-owned hook for it? ignore-scripts is good w.r.t security and readability